### DISCUSSION TITLE
2 instances Z2M on the same server (Ubuntu linux without docker)

### DISCUSSION THREAD

**Discussion by steflp** [Koenkk/zigbee2mqtt#18889] [General]
Hello 
I  dont use Home Assistant ( it is important) 
I  dont use Docker  too  !
-----------------------------
I  have installed first Z2M with usb key SonOff  and it s OK 
I  must install a second z2M on the same server  with another key ( SLZB-06 Zigbee  lan /wifi) 
Is it possible to do this ? and  how ? 

thanks a lot  ! 

Stef

**Comment by francisp2** [comment]
2 times install
Zigbee2MQTT 1 in /opt/zigbee2mqtt
Zigbee2MQTT2 in /opt/zigbee2mqtt2

- create the service twice, with another name
- use different mqtt topic and panid in zigbee2mqtt2
-

**Comment by steflp** [comment]
Ok thanks  ! 
I dont remember that  I can choose directory  ;-(  ... so it will be more easy  (just modify systemctl (it will be easy I hope  !)
